packages/bull-vault/src/LeverageBull.sol

```diff
@@ -59,13 +59,12 @@ contract LeverageBull {
      * @param _ethUsdPrice ETH price in USDC
      * @return ETH deposited as collateral in Euler and borrowed amount of USDC
      */
-    function _leverageDeposit(uint256 _crabAmount, uint256 _bullShare, uint256 _crabPrice, uint256 _ethUsdPrice)
-        internal
+    function calcLeverageEthUsdc(uint256 _crabAmount, uint256 _bullShare, uint256 _crabPrice, uint256 _ethUsdPrice)
```

Comment:
Would prefer keeping the code in the same block if we are not calling it in different places, easier to read and audit + less expensive as we don't need to call an internal function

Reply:
Was thinking about calling `calcLeverageEthUsdc` from `FlashBull`. Thoughts on best way to do that w/o repetitive code?

Reply:
Also let's keep the functions in order, external -> public -> internal -> private (view->pure last) https://docs.soliditylang.org/en/v0.8.14/style-guide.html#order-of-functions